### PR TITLE
Blogger: Spec controllers

### PR DESCRIPTION
**Pull request by lawrahkonwea**
### Hi Reviewer....
hello

### In this pull request I:

- [ ]  Added RSpec as a gem to the project.

- [ ] Created a Request spec file for all the controllers.

- [ ]  Made sure that for each action to check:

- If response status was correct.

- If a correct template was rendered.

- If the response body includes correct placeholder text.

Have a great day!

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -1,7 +1,27 @@
 require 'rails_helper'
 
 RSpec.describe 'Posts', type: :request do
```
Comment (gemmen29) [inline spec/requests/posts_spec.rb:3]: - [ ] Kindly add a test to check if the response body includes correct placeholder text as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=If%20the%20response%20body%20includes%20correct%20placeholder%20text.) for the `post#index` and `post#show`.

File: spec/requests/users_spec.rb [hunk 2]
```diff
@@ -2,6 +2,36 @@
...
+    end
+
+    it 'renders the correct template' do
+      get '/users'
+      expect(response).to render_template('index')
+    end
+
+    it 'should include correct placeholder' do
+      get '/users'
+      expect(response.body).to include('')
```
Comment (gemmen29) [inline spec/requests/users_spec.rb]: - [ ] Kindly fix this placeholder test for `user#index` with the correct placeholder text as it now only checks if the `response.body` contains an empty string.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @lawrahkonwea, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/users_spec.rb (hunk 2 above)
Comment (lawrahkonwea) [reply spec/requests/users_spec.rb]: @gemmen29 I would do that now. Thank you

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @lawrahkonwea , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] `Rspec` testing gem is used to test controllers :white_check_mark: 
- [x] All controller specs are running correctly :white_check_mark: 
- [x] Pull request has a descriptive title and summary :white_check_mark: 
- [x] Repository has a descriptive readme file :white_check_mark: 
- [x] Git flow is used correctly :white_check_mark:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**